infra/postsubmit/proto/postsubmit.proto

```diff
@@ -0,0 +1,42 @@
+syntax = "proto3";
+
+package infra.postsubmit;
+
+message Build {
+  // Name of this build. Needs to be unique among all builds, and should match
+  // the regex `[a-z][a-z_]+`.
+  string name = 1;
```

Comment:
So I don't like using names here, considering there's a high chance of collisions between teams. Why not use path relative from git root?

Reply:
This makes sense if this message is only supposed to be used from singular config files (which is the current plan), but if this ever changes (nesting this into a higher-level message, having multiple Builds in a file, etc) then one would really want the name field back, since keying off filename no longer makes sense.

Path from git root would also create potentially noisy/non-intuitive names - this forces teams to name their builds, meaning no intuition/translation is necessary (though probably there will be some repetition of the file's basename in this field)